frontend/src/components/ImagePicker.js

```diff
@@ -197,35 +197,45 @@ export default class ImagePicker extends Component {
 
   lookupSocialMediaAvatars(website, twitter)
   {
-    const { profileForm, newUser, getSocialMediaAvatars } = this.props;
+    const { profileForm, newUser, getSocialMediaAvatars, uploadOptionFirst } = this.props;
+    const defaultPresetIndex = uploadOptionFirst ? 1 : 0;
 
-    if (!this.state.isLoading)
-    {
+    if (!this.state.isLoading) {
       this.setState({isLoading: true});
     }
 
     getSocialMediaAvatars(newUser.id, {website: website, twitterHandle: twitter, name: profileForm.name})
     .then((response) => {
       const currentOption = this.options[this.state.currentIndex];
       const results = response.json.filter((option) => this.blacklist.indexOf(option.src) === -1 );
+      const newResults = results.filter((option) => {
+        let alreadyExists = false;
```

Comment:
Is there a benefit of using `let` vs `var`? If not, could you change to `var`? It's used everywhere else in the codebase.